.github/workflows/autopr.yaml

```diff
@@ -0,0 +1,44 @@
+name: Manual workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      majorVersion:
+        description: 'Major Version e.g. 3.0'
+        required: true
+      targetVersion:
+        description: 'Target Version e.g. 3.0.151471'
+        required: true
+
+jobs:
+  createPR:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: dev
+    - id: createBranch
+      run: |
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+        cd host
+        ./update-version.sh ${{ github.event.inputs.majorVersion }} ${{ github.event.inputs.targetVersion }}
+        git add .
+        git commit -m "Update Version ${{ github.event.inputs.targetVersion }}"
+    - name: Create PullRequest
+      id: createPullRequest
+      uses: peter-evans/create-pull-request@v3
+      with:
+        title: Update to verison ${{ github.event.inputs.targetVersion }}
+        body: |
+          - Update ${{ github.event.inputs.majorVesion}} to ${{ github.event.inputs.targetVersion }}
+        branch: action/release.${{ github.event.inputs.targetVersion }}
+        labels: |
+          automated pr
+        reviewers: tsuyoshiushio
```

Comment:
>tsuyoshiushio [](start = 19, length = 13)

Add other reviewers?

Reply:
Absolutely! This pr is for testing. ( we need to test it first). So that I just added me at first.

Reply:
Thanks for clarifying ! if this is just for testing, might be worth checking this in, testing this out and then make updates on this later if we're missing anything.

---
In reply to: [599829115](https://github.com/Azure/azure-functions-docker/pull/410#discussion_r599829115) [](ancestors = 599829115)